devices/ePaper/Drivers/IePaperDisplay.cs

```diff
@@ -0,0 +1,123 @@
+﻿// Copyright (c) 2022 The nanoFramework project contributors
```

Comment:
the datasheet file should not be checked in because it is under copyrights. Please have a link on it rather than checking it in.

Reply:
Makes perfect sense! Thanks for pointing this out. I've added a link to the datasheet in the `README` file.